blockchain/types/account/account_serializer.go

```diff
@@ -127,3 +147,29 @@ func (ser *AccountSerializer) UnmarshalJSON(b []byte) error {
 
 	return json.Unmarshal(dec.Account, ser.account)
 }
+
+// UnextendRLP unextends ExtHash fields within an RLP-encoded account.
+// If the supplied bytes is not an RLP-encoded account, or does not contain any ExtHash,
+// then return the supplied bytes unchanged.
+func UnextendRLP(b []byte) []byte {
```

Comment:
The input and output is not RLP-encoded, but serialized account, right?
How about renaming the function to `UnextendSerializedAccount`?

Reply:
Good suggestion. Fixed in 442ba378.